widgets/tabs/js/tabs.js

```diff
@@ -52,7 +52,7 @@ jQuery( function ( $ ) {
 				}
 			};
 			
-			$tabs.click( function () {
+			$tabs.on('keyup click', function () {
 				selectTab( this );
```

Comment:
This automatically selects the tab as soon as the Tab button is released, which isn't usual Tab button behavior. Pressing Tab should cycle focus to the tab and then it should only be selected when the Enter key is pressed.